cisco_secure_email_threat_defense/README.md

```diff
@@ -1,43 +1,63 @@
 
-### Installation
+**Disclaimer**: Your use of this integration, which may collect data that includes personal information, is subject to your agreements with Datadog. Cisco is not responsible for the privacy, security or integrity of any end-user information, including personal data, transmitted through your use of the integration.
 
-The Cisco Secure Email Threat Defense check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+## Setup
 
 ### Configuration
 
-!!! Add list of steps to set up this integration !!!
+#### Get API Credentials for Cisco Secure Email Threat Defense 
+
+
+#### Steps to generate API key, Client ID and Client Password:
+1. Log in to the Cisco Secure Email Threat Defense UI.
+2. Navigate to **Administration** and select the **API Clients** tab.
+3. Click on **Add New Client**.
+4. Enter a **Client Name** and an optional description, then click **Submit**. This generates your Client ID and Password.
+    **Note**: Be sure to save the Client Password, as it will only be displayed once.
+5. Retrieve the API key from the **API Key** section.
 
-### Validation
+#### Cisco Secure Email Threat Defense Datadog Integration Configuration
 
-!!! Add steps to validate integration is functioning as expected !!!
+Configure the Datadog endpoint to forward Cisco Secure Email Threat Defense logs to Datadog.
+
+1. Navigate to **Cisco Secure Email Threat Defense**.
+2. Add your Cisco Secure Email Threat Defense credentials and verdict delay.
+
+| Cisco Secure Email Threat Defense Parameters | Description  |
+| -------------------- | ------------ |
+| Host Name               | The Host Name for Cisco Secure Email Threat Defense.|
+| API Key           | API Key from Cisco Secure Email Threat Defense.         |
+| Client ID      | Client ID from Cisco Secure Email Threat Defense.    |
+| Client Password      | Client Password from Cisco Secure Email Threat Defense.    |
+| Verdict Delay      | Time (in minutes) required to allow verdicts to be updated. Specify a number from 0 to 720. The Default value is 30.    |
 
 ## Data Collected
 
-### Metrics
+### Logs
 
-The Cisco Secure Email Threat Defense does not include any metrics.
+The Cisco Secure Email Threat Defense integration collects and forwards Cisco Secure Email Threat Defense message logs to Datadog. This integration will ingest messages with verdict values of scam, malicious, phishing, BEC, spam, graymail, and neutral.
 
-### Service Checks
+**Note**: Events are fetched with a delay according to the time specified in the Verdict Delay. This delay is intended to accommodate retrospective verdicts. However, this does not guarantee that all retrospective verdicts are processed within this timeframe, as the time required for updates varies.
```

Comment:
Isn't the whole point that the user specifies the interval, thus we can't be specific about it here? Or am I missing some nuance of how this works?

Reply:
@neko-dd 
According to Jason's comment, he mentions that the data will be collected in every xx minutes interval, and it will be collected till the current time minus verdict delay (which is specified by the user). I hope this clears your doubt.

@jnhunsberger 
I have updated the sentence as per your suggestion. 
Let me know if it has to be changed.